tldraw/tldraw
# feat(translations): add Danish translation

Added Danish translation to the tldraw package.

### Change type


### Test plan

1. Switch language to Danish in the editor settings.
2. Verify that UI elements are correctly translated.


### Release notes

- Added Danish translation
